web/turkey/survey/static/survey/js/auditors/clicks_total.js

```diff
@@ -0,0 +1,25 @@
+var AuditorClicksTotal = {
+    clicks_total: 0,
+    log_click_event: function (e) {
+        e.stopPropagation();
```

Comment:
On the same note as in the previous note, a click will be registered for every DOM element underneath the topmost element. So if we have <body><div></div></body>, clicking the div would also click the body.